docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,86 @@
+Akka HTTP Quickstart with Scala
+===============================
+
+The Akka HTTP modules implement a full server- and client-side HTTP stack on top of akka-actor and akka-stream.  It’s not a web-framework but rather a more general toolkit for providing and consuming HTTP-based services.  While interaction with a browser is of course also in scope, it is not the primary focus of Akka HTTP.
+
+Akka HTTP follows a rather open design and many times offers several different API levels for "doing the same thing."  The users get to pick the API level of abstraction that is most suitable for their applications.  This approach means that, if you have trouble achieving something using a high-level API, there’s a good chance that you can get it done with a low-level API, which offers more flexibility but might require you to write more application code.
+
+This Hello World example demonstrates some Akka HTTP fundamentals. Within 30 minutes, you should be able to download, run the sample application, and use this documentation to guide you through this example's building blocks.
+
+## Prerequisite
+
+Having a basic understanding of Akka actors will help the reader of this guide. There is a [getting started guide](https://developer.lightbend.com/guides/akka-quickstart-scala/) for Akka actors should you feel like brushing up your knowledge thereof.
```

Comment:
s/getting started guide/Quickstart guide/

there is a section in the full documentation that is the full Getting Started guide